src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -38,7 +42,8 @@ internal partial class ProjectState
             ImmutableDictionary<DocumentId, DocumentState> documentStates,
             ImmutableDictionary<DocumentId, TextDocumentState> additionalDocumentStates,
             AsyncLazy<VersionStamp> lazyLatestDocumentVersion,
-            AsyncLazy<VersionStamp> lazyLatestDocumentTopLevelChangeVersion)
+            AsyncLazy<VersionStamp> lazyLatestDocumentTopLevelChangeVersion,
+            ValueSource<ProjectStateChecksums> lazyChecksums)
```

Comment:
The only code in the body you added was: ```_lazyChecksums = new AsyncLazy<ProjectStateChecksums>(ComputeChecksumsAsync, cacheResult: true);```  This does not use ```lazyChecksum```.  So i don't get what the point of the parameter is.